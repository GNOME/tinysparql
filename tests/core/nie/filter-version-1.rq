SELECT ?version
WHERE { ?x nie:version ?version }
ORDER BY ?version
