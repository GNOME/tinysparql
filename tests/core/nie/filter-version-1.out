"stringly data for nie:version"
